content/08.methods.md

```diff
@@ -30,6 +30,37 @@ The ability to irrefutably prove manuscript existence at a past time could be im
 
 ### Author contributions
 
+#### Version 2.0
+
+We continued using the open repository on the GitHub version control platform ([`greenelab/deep-review`](https://github.com/greenelab/deep-review)) [@url:https://github.com/greenelab/deep-review], which was established to write the Version 1.0 manuscript.
+Drafted multiple sub-sections along with extensive editing, pull request reviews, or discussion:
+Drafted one or more sub-sections: .
+Revised specific sub-sections or supervised drafting one or more sub-sections: .
+Drafted sub-sections, edited the manuscript, reviewed pull requests, and coordinated co-authors: C.S.G.
+
+{% for v2, authors in manubot.authors | groupby('v2') %}
```

Comment:
I think this `groupby` is what is causing the build errors.  The changes in 65bfcc9 may not be needed?